pandas/io/formats/format.py

```diff
@@ -1110,7 +1121,7 @@ def format_array(
     space
     justify
     decimal
-    leading_space : bool, optional
+    leading_space : bool, default is 'compat'
         Whether the array should be formatted with a leading space.
```

Comment:
same comment as previous here what is the intended type of this?

Reply:
yeah, there was no type hinting back then, and I didn't notice this so just copy paste the change in my previous PR. my bad, changed.